ui/src/components/VoxPrompts.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Can we see screenshots of this new component in the PR descriptions? :)

Reply:
will do in the future as I showed this already in podsync